pkg/handlers/v1/hydrator.go

```diff
@@ -59,6 +60,17 @@ func (h *HydrationHandler) Handle(ctx context.Context, evt AssetEvent) error {
 	}
 	assetVulnEvent := domainAssetVulnerabilityDetailsToEvent(assetWithVulnerabilityDetails)
 	_, err = h.Producer.Produce(ctx, assetVulnEvent)
+	switch err.(type) {
+	case producer.ErrSizeLimitExceeded:
+		logger.Error(logs.PayloadSizeLimitExceededError{
```

Comment:
Question: Just want to make I understand the impact of this change. We're still reporting this as an error, but we're also logging the the PayloadSizeLimitExceeded error so we have more insight as to what's going on. And since the alerts for these errors were fixed, we'll now get alerted if we get a lot (at least 50) of these PayloadSizeLimitExceeded errors within a 5 minute period. I'm fine if this is the case since we'll talk about it in TechOps and the alerts will make sure we prioritize this issue.

Reply:
In our case, regardless of making this change, the producer is returning an error, so we would get alerted with our current config. That's more of an internal detail of how we run this. This change primarily logs additional context so we can identify problematic assets easier.